public/css/user/request.css

```diff
@@ -1,3 +1,121 @@
 
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+  }
+  .main{
+    margin-top: 85px;
+  }
+  .nav--form {
+    display: flex;
+    justify-content: center
+  }
+  .nav--form-personalDetails {
+    
```

Comment:
fix indentation

Reply:
I did it can you make a review?

Reply:
see [another requested cahnge](https://github.com/FACG6/enhance-a-smile/pull/102#pullrequestreview-217138723)